### PR TITLE
[Darkside] Switch CSS update

### PR DESCRIPTION
**Pull request by KenAJoh**
Felt doing more nesting for readonly state would make it less readable, so left it as is

### CODE REVIEW

File: @navikt/core/css/darkside/form/switch.darkside.css [hunk 1]
```diff
@@ -181,12 +158,13 @@
   cursor: not-allowed;
 }
 
-/* Readonly */
+/* -------------------------- Switch Readonly state ------------------------- */
 .navds-switch--readonly > .navds-switch__track,
```
Comment (KenAJoh) [inline @navikt/core/css/darkside/form/switch.darkside.css]: Consider adding all this nested inside .navds-switch--readonly to avoid repitition

**Comment by KenAJoh** [comment]
TODO:
- Bg-input in OFF-state
- 18px thumb in OFF-state
- Update padding on off-state
- Update thumb color on off-state

File: @navikt/core/css/darkside/form/switch.darkside.css
Comment (HalvorHaugan) [inline @navikt/core/css/darkside/form/switch.darkside.css:1]: Not important, but the alignment is a bit off in forced-colors mode
image

Comment (KenAJoh) [reply @navikt/core/css/darkside/form/switch.darkside.css:1]: Think its fixed now

Comment (HalvorHaugan) [inline @navikt/core/css/darkside/form/switch.darkside.css:1]: One last thing: Should disabled switch in ON state have green border in forced-colors mode?
image

Comment (KenAJoh) [reply @navikt/core/css/darkside/form/switch.darkside.css:1]: Fixed 🔨

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**